modules/navigation.js

```diff
@@ -0,0 +1,14 @@
+const links = document.querySelectorAll('.link');
+const slides = document.querySelectorAll('.slide');
+
+export default function navigate() {
+  links.forEach((link, id) => {
+    link.addEventListener('click', (e) => {
+      e.preventDefault();
+      slides.forEach((slide) => {
+        slide.classList.add('disable');
```

Comment:
- [ ] Same here, Please, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)